src/frontend/efiling-frontend/src/domain/package/package-review/PartyList.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { formatFullName } from "../../../modules/helpers/StringUtil";
+import "./PartyList.scss";
+
+export default function ParyList({ parties }) {
+  return (
+    <div className="ct-party-list">
+      <div className="header">
+        <span className="d-none d-lg-inline col-lg-6">Name</span>
+        <span className="d-none d-lg-inline col-lg-3">Party Role</span>
+        <span className="d-none d-lg-inline col-lg-3">Party Type</span>
+      </div>
+      <ul>
+        {parties &&
+          parties.map((party) => (
+            <li key={party.firstName}>
```

Comment:
@yousiefc , what's a good strategy here? React needs a unique identifier foreach <li>, however none is available.
Perhaps we can take a hash of the entire object?

Reply:
You can pass `index` in as a second parameter of the map callback and use that it would be unique

Reply:
hmm. lint error "do not use index as a key".
hash(party) worked, though.